src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/reporting/ReportingParserTest.kt

```diff
@@ -0,0 +1,148 @@
+            {"id": "E4", "type": "MoveRedirect", "actioned_by": "geoamey", "notes": null, "created_at": "2020-08-03 15:43:12 +0100", "updated_at": "2020-08-03 15:43:12 +0100", "eventable_id": "M1", "eventable_type": "move", "occurred_at": "2020-06-16T10:20:30+01:00", "recorded_at": "2020-06-16T10:20:30+01:00", "details": {"to_location": "BRSTCC"}}
+        """.trimIndent()
+
+        return listOf(report1, report2)
+    }
+
+    fun journeyReports() : List<String>{
+        val report1 = """
+        {"id":"J1","move_id":"M1","billable":false,"state":"completed","client_timestamp":"2020-09-08T12:49:00+01:00","created_at":"2020-09-07T15:00:58+01:00","updated_at":"2020-09-07T15:00:58+01:00","from_location_type":"probation_office","from_location":"GCS11","to_location_type":"probation_office","to_location":"HPS008","supplier":"geoamey","vehicle_registration":"UHE-92"}
+        {"id":"J2","move_id":"M2","billable":false,"state":"completed","client_timestamp":"2020-09-08T06:55:00+01:00","created_at":"2020-09-07T15:01:01+01:00","updated_at":"2020-09-07T15:01:01+01:00","from_location_type":"court","from_location":"PRSTYC","to_location_type":"probation_office","to_location":"MRS023","supplier":"serco","vehicle_registration":"BNX-76"}
+    """.trimIndent()
+
+        // First journey has same id, Second journey has the same move id, 3rd journey is for a different move
+        val report2 = """
+        {"id":"J1","move_id":"M1","billable":true,"state":"completed","client_timestamp":"2020-09-08T12:49:00+01:00","created_at":"2020-09-07T15:00:58+01:00","updated_at":"2020-09-07T15:00:58+01:00","from_location_type":"probation_office","from_location":"GCS11","to_location_type":"probation_office","to_location":"HPS008","supplier":"geoamey","vehicle_registration":"UHE-92"}
+        {"id":"J3","move_id":"M2","billable":false,"state":"completed","client_timestamp":"2020-09-08T06:55:00+01:00","created_at":"2020-09-07T15:01:01+01:00","updated_at":"2020-09-07T15:01:01+01:00","from_location_type":"court","from_location":"PRSTYC","to_location_type":"probation_office","to_location":"MRS023","supplier":"serco","vehicle_registration":"BNX-76"}
+        {"id":"J4","move_id":"M3","billable":false,"state":"completed","client_timestamp":"2020-09-08T06:55:00+01:00","created_at":"2020-09-07T15:01:01+01:00","updated_at":"2020-09-07T15:01:01+01:00","from_location_type":"court","from_location":"PRSTYC","to_location_type":"probation_office","to_location":"MRS023","supplier":"serco","vehicle_registration":"BNX-76"}
+    """.trimIndent()
+
+        return listOf(report1, report2)
+    }
+
+    @Test
+    fun `Get import moves`() {
+        val moves = ReportingParser.parseAsMoves(moveReports())
+        //There should be 4 moves because the first one in both files is the same
+        Assertions.assertEquals(4, moves.size)
```

Comment:
So not usering assertJ?  Not of any mandate on this but it is a tool quite often used in DPS (and my me).  A more fluent inferface, blah blah.